bloch_sphere/animate_bloch.py

```diff
@@ -419,8 +445,12 @@ def run_from_command_line():
         'Save an mp4 video instead of a GIF')
     parser.add_argument('--fps', type=float, default=20, help=
         'Sets the animation frame rate')
+    parser.add_argument('--style', type=str, choices=['sphere', 'arrows'], 
+        default='sphere', help='Different style of showing rotated bloch sphere. '
+        'Supported options are [sphere, arrows].')
```

Comment:
`bloch_sphere --help` should automatically generate a description of the available choices.
```suggestion
```

Reply:
Updated according to later comments